source/geod24/concurrency.d

```diff
@@ -947,6 +948,15 @@ class FiberScheduler
         dispatch();
     }
 
+    /**
+    * This commands the scheduler to shut down at the end of the program.
```

Comment:
What's the expectation here ?
The comment seems incorrect. At the end of the program, the scheduler would have already exited.
And this does not *force* the scheduler to end immediately, but only at the next `yield`.

Reply:
What do you think about this?This terminates function `dispatch` even if there is a running fiber.

Reply:
No, if there is a running fiber, and the running fibers does not yield, it will not terminate.

Reply:
I'm sorry. I just realized I don't need this function.